src/condor_credd/condor_credmon_oauth/condor_vault_storer

```diff
@@ -81,6 +81,13 @@ fi
 #  sure that credmon has a long-duration one, but copy it to $VTOKEN if a
 #  new one is generated.
 ID="`id -u`"
+
+CREDKEY=""
+IS_POMS_ANALYSIS=false
```

Comment:
It would be better to not have code with the name POMS in condor_vault_storer.  It would be better to have generic capabilities that might at other points be useful in other use cases.